pkg/ccl/backupccl/show.go

```diff
@@ -82,6 +82,8 @@ func (m manifestInfoReader) header() colinfo.ResultColumns {
 	return m.shower.header
 }
 
+//showBackup reads backup info from the manifest, populates the manifestInfoReader,
```

Comment:
nit: space between // showBackup

nit: period at the end of the comment.

Reply:
done. question: this latest commit passed the full linter on github and bazel, which i guess means the linter doesn't catch this stuff? does your file watcher catch this?

Reply:
i guess not... I do think my file watcher automatically adds these spaces.